packages/crypto-wallet-core/src/derivation/eth/index.ts

```diff
@@ -28,7 +28,7 @@ export class EthDeriver implements IDeriver {
     const x = ecPoint.getX().toBuffer({ size: 32 });
     const y = ecPoint.getY().toBuffer({ size: 32 });
     const paddedBuffer = Buffer.concat([x, y]);
-    const address = utils.keccak256(paddedBuffer).slice(26);
+    const address = utils.keccak256(`0x${paddedBuffer.toString('hex')}`).slice(26);
```

Comment:
utils.keccak256 param takes in `typeof 'string | BN'`

Cannot pass in type `Buffer`

Reply:
https://github.com/ethereum/web3.js/blob/a5e832fc20088cf4c4e916bc0ec3107ceb4428d0/packages/web3-utils/types/index.d.ts#L93

Reply:
This change seems to be covered but test "should be able to generate a valid ETH address" and "should be able to generate a valid ETH address, privKey, pubKey".  I think we need to include more cases, including edge cases, on the test suite, on a future PR.